tools/lint.py

```
@@ -3353,6 +3363,292 @@ def run(root: Path) -> list[str]:
+            if (isinstance(node.value, ast.Constant) and _is_marker(node.value.value)
+                    and isinstance(node.target, ast.Name)):
+                names.add(node.target.id)
+            continue
+        if not isinstance(node, ast.Assign):
+            continue
+        for target in node.targets:
+            if isinstance(target, ast.Name) and isinstance(node.value, ast.Constant):
+                if _is_marker(node.value.value):
+                    names.add(target.id)
+            elif isinstance(target, ast.Tuple) and isinstance(node.value, ast.Tuple):
+                for name, value in zip(target.elts, node.value.elts):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)